src/user/views/adsManager/views/advanced/components/campaign/fields/BudgetField.tsx

```diff
@@ -1,16 +1,42 @@
 import { Box, Divider, InputAdornment, Stack, Typography } from "@mui/material";
+import { FormikTextField } from "../../../../../../../../form/FormikHelpers";
+import React, { useEffect } from "react";
+import { useField, useFormikContext } from "formik";
+import { CampaignForm } from "../../../../../types";
+import { differenceInDays } from "date-fns";
 import {
-  FormikRadioControl,
-  FormikRadioGroup,
-  FormikTextField,
-} from "../../../../../../../../form/FormikHelpers";
-import React from "react";
+  MIN_PER_CAMPAIGN,
+  MIN_PER_DAY,
+} from "../../../../../../../../validation/CampaignSchema";
```

Comment:
Feel free to set [baseUr](https://www.typescriptlang.org/tsconfig#baseUrl) like we do in ads admin to avoid all those `..`'s.  Not needed as part of this PR.